lib/api/controllers/securityController.js

```diff
@@ -0,0 +1,79 @@
+var
+  ResponseObject = require('../core/models/responseObject');
+
+module.exports = function SecurityController (kuzzle) {
+
+  /**
+   * Get a specific role according to the given id
+   * @param {RequestObject} requestObject
+   * @returns {Promise}
+   */
+  this.getRole = function (requestObject) {
+    kuzzle.pluginsManager.trigger('data:getRole', requestObject);
```

Comment:
Question: I wonder if plugin hooks on API security routes shouldn't be "security:..." instead of "data:..."